alpha/apps/kaltura/lib/storage/urlTokenizers/kWowzaSecureTokenUrlTokenizer.php

```diff
@@ -0,0 +1,157 @@
+<?php
+
+class kWowzaSecureTokenUrlTokenizer extends kUrlTokenizer
+{
+	
+	/**
+	 * @var string
+	 */
+	protected $paramPrefix;
+	
+	/**
+	 * @var bool
+	 */
+	protected $shouldIncludeClientIp;
```

Comment:
better use the existing kUrlTokenizer/KalturaUrlTokenizer::limitIpAddress field

Reply:
I can do that but from what I can tell KalturaUrlTokenizer::limitIpAddress is intended to enforce IP based ACLs, via Access Control Profile I assume, which is not what this option is doing. This option includes the Client IP address in the hash generation like [Vnpt tokenzier](https://github.com/jbrule/server/blob/85031093e3e3f591b956d641781ffbafae47f5d5/alpha/apps/kaltura/lib/storage/urlTokenizers/kVnptUrlTokenizer.php)

--Josh

Reply:
No, it has the exact same meaning as you have here - it determines whether the token is signed with the client ip address or not. You can see for example - kCloudFrontUrlTokenizer::generateToken